deploy/assisted-image-service.yaml

```diff
@@ -34,6 +34,7 @@ items:
               httpGet:
                 path: /health
                 port: 8080
+              initialDelaySeconds: 30
```

Comment:
I'm not sure if it's enough for the amount of images downloaded on PSI. It took at least a few minutes to download all of them and enable the API

Reply:
ah I didn't your comment in the PR description. sorry about it